components/browser/session/src/test/java/mozilla/components/browser/session/engine/EngineObserverTest.kt

```diff
@@ -62,6 +68,10 @@ class EngineObserverTest {
                 return emptyMap()
             }
 
+            override fun loadData(data: String, mimeType: String) {
+                // TODO: What do we do here?
```

Comment:
If it's not needed in the test, we can just leave it empty. Did you want to keep that comment?

Reply:
This was my second question about testing but I figured it out; forgot to remove this todo. :)